map/bookmark_manager.cpp

```diff
@@ -309,6 +309,10 @@ Track * BookmarkManager::GetTrackForEdit(kml::TrackId trackId)
   if (it == m_tracks.end())
     return nullptr;
 
+  auto const groupId = it->second->GetGroupId();
+  if (groupId != kml::kInvalidTrackId)
+    m_changesTracker.OnUpdateLine(trackId);
```

Comment:
What is the point of calling OnUpdateLine when a track is retrieved for edit instead of calling it after it was edited?